src/components/CardHeadline/styled.js

```diff
@@ -1,7 +1,8 @@
 import styled from 'styled-components';
 
 const StyledHeadline = styled.h2`
-	padding: 20px;
+	position: absolute;
```

Comment:
Like I mentioned above. I guess there is a more simple way to do it without using positioning. 🙂